wcivf/apps/elections/templates/elections/includes/_post_meta_description.html

```diff
@@ -1 +1,7 @@
-{% autoescape off %}See all {{ person_posts|length }} candidates in {{ object.post.label }} in the {{ object.election.name }} on {{ object.election.election_date }}: {% for pp in person_posts %}{{ pp.person.name|safe }} ({{ pp.party.party_name }}) {% endfor %}{% endautoescape %}
\ No newline at end of file
+{% if postelection.people|length %}
+{% autoescape off %}See all {{ postelection.people|length }} candidates in the {{ object.election.name }} on {{
```

Comment:
Really minor thing but I find it easier to read when these are indented (as done in the `_post_meta_title.html`)

Reply:
Very strange but on save, vs code would not accept my indentation changes. I'll look into it but didn't want to hold this up any longer than I have already. Nevertheless, appreciate the reminder 👍